src/shell/meanbee/configpoweredcss/regenerate.php

```diff
@@ -0,0 +1,48 @@
+<?php
+require_once __DIR__ . '/../../abstract.php';
+
+class Meanbee_Configpoweredcss_Regenerate extends Mage_Shell_Abstract
+{
+
+    /**
+     * Run script
+     *
+     */
+    public function run()
+    {
+
+        /**
+         * Shell script regenerate the configpowered css for all stores
+         */
+
+        try {
+
+            /** @var Meanbee_ConfigPoweredCss_Model_Css $css */
+            $css = Mage::getModel('meanbee_configpoweredcss/css');
+
+            $stores = Mage::app()->getStores();
+            foreach ($stores as $storeId => $store) {
+                try {
+                    Mage::app()->setCurrentStore($storeId);
+                    $css->publish($storeId);
+                } catch (Exception $e) {
+                    Mage::logException($e);
+                    return;
+                }
+            }
+
+            fwrite(STDERR,
+                "Config powered css has been regenerated successfully.\n");
+            exit(0);
+        } catch (Exception $e) {
+            Mage::logException($e);
+
+            fwrite(STDERR,
+                "There was an error when regenerating the config powered css, it has been logged.\n");
```

Comment:
It would be better to show the error to the user, instead of making them jump through hoops for it (logging into the server, navigating to the log directory, finding the specific error in the endless stream of `exception.log`).